concepts/observability/README.md

```diff
@@ -36,4 +36,5 @@ The observability tools listed below are ones that have been tested to work with
 ### Distributed Tracing
 
 * [How-To: Set up Zipkin](../../howto/diagnose-with-tracing/zipkin.md)
-* [How-To: Set up Application Insights](../../howto/diagnose-with-tracing/azure-monitor.md)
\ No newline at end of file
+* [How-To: Set up Application Insights with Local Forwarder](../../howto/diagnose-with-tracing/local-forwarder.md)
```

Comment:
@msfussell do we want to remove local forwarder doc here? otherwise, I would add `[Deprecated]` comment on this link.